casts/highlights/src/useOnWheelHandler.ts

```diff
@@ -1,13 +1,33 @@
 /* eslint-disable no-param-reassign */
 import * as React from 'react';
+import { I18nManager, NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
 import { nanoid } from 'nanoid/non-secure';
 
 import { getWorkletFromParentHandler } from '@tonlabs/uikit.scrolls';
-import type { NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
+import { clamp } from 'lodash';
```

Comment:
```suggestion
import clamp from 'lodash/clamp';
```

in order to make the final bundle smaller